pom.xml

```diff
@@ -10,7 +10,7 @@
   <groupId>org.folio</groupId>
   <artifactId>mod-lists</artifactId>
   <name>mod-lists</name>
-  <version>1.0.0</version>
+  <version>1.0.2-SNAPSHOT</version>
```

Comment:
There's a conflict here, so I suspect I'll need to disable PR merge checks and merge it manually to keep it from messing up our version on `master`